src/Runner.Common/JobServerQueue.cs

```diff
@@ -804,6 +822,34 @@ private async Task UploadSummaryFile(SummaryUploadFileInfo file)
                 }
             }
         }
+
+        private async Task UploadResultsStepLogFile(ResultsUploadFileInfo file)
+        {
+            bool uploadSucceed = false;
+            try
+            {
+                Trace.Info($"Starting to step log file to results service {file.Name}, {file.Path}");
+                var cancellationTokenSource = new CancellationTokenSource();
+                await _jobServer.CreateResultsStepLogAsync(file.PlanId, file.JobId, file.RecordId, file.Path, file.Finalize, file.FirstBlock, file.TotalLines, cancellationTokenSource.Token);
+
+                uploadSucceed = true;
+            }
+            finally
+            {
+                if (uploadSucceed && file.DeleteSource)
+                {
+                    try
+                    {
+                        File.Delete(file.Path);
+                    }
+                    catch (Exception ex)
+                    {
+                        Trace.Info("Catch exception during delete success results uploaded step log file.");
```

Comment:
```suggestion
                        Trace.Info("Exception encountered during deletion of already successfully uploaded step log file to results.");
```

Was having a had time understanding this. We're effectively deleting a temporary file from my understanding